components/nats-init/Makefile

```diff
@@ -0,0 +1,11 @@
+APP_NAME = nats-init
+APP_PATH = components/$(APP_NAME)
+BUILDPACK = eu.gcr.io/kyma-project/test-infra/buildpack-golang-toolbox:v20190930-d28d219
```

Comment:
(nit) If we only use `docker.mk`, is `BUILDPACK` and `APP_PATH` still required? From my understanding only `APP_NAME` and `SCRIPTS_DIR` is required.